fixtures/legacy-jsx-runtimes/react-14/cjs/react-jsx-runtime.development.js

```diff
@@ -103,7 +103,7 @@ function printWarning(level, format, args) {
     }
 
     var argsWithFormat = args.map(function (item) {
-      return '' + item;
+      return ''.concat(item);
```

Comment:
it's worth noting that using concat means that react will start breaking if someone does `delete String.prototype.concat`; whereas using `String` would only break if someone did the much less likely `delete window.String` (and caching `String` somewhere would prevent this entirely)

Reply:
Changing the code in this PR from `''.concat(` to `String(` would be a trivial search and replace operation, and I'd be happy to do it. 

@bvaughn - I've learned enough while building this PR that I now know how to disable the lint rule that disallows `String(x)`, but I'm not sure why that rule is there in the first place and whether FB would be OK with it being disabled. Do you know?

FWIW, I agree with @ljharb that `String(x)` would be better here. It also may bypass https://github.com/facebook/flow/issues/8728 which is currently blocking this PR from passing its Flow tests.

Reply:
BTW, I found the original PR that added this no-`String()` lint rule: #9082.

Reply:
> it's worth noting that using concat means that react will start breaking if someone does `delete String.prototype.concat`; whereas using `String` would only break if someone did the much less likely `delete window.String` (and caching `String` somewhere would prevent this entirely)

This seems like a pretty unlikely scenario and one that we don't really try to support with any other prototype methods<sup>1</sup>. (We often close over global APIs e.g. `window.performance` or `window.setTimeout` but generally not methods on prototypes.)

<sup>1</sup> The one exception is, I believe, `Object.prototype.hasOwnProperty`.